### PR TITLE
reduce max_old_space_size for jest tests

### PR DESCRIPTION
**Pull request by peterbe**
### Why:

Experimenting to see if the `jest tests/rendering` gets better. 

### Check off the following:

- [ ] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by peterbe** [comment]
@mikesurowiec It didn't seem to be the silver bullet we had hoped for. But it still seems like the totally right thing to do.

**Review by heiskr — DISMISSED** [review]

**Comment by peterbe** [comment]
@mikesurowiec Now it's just changing the memory amount. 
Still, it doesn't appear to solve much. Just a good practice.

**Comment by MAC-ARTHUR-TRUE** [comment]
Madison chambers or Chamberlin POW MIA Afghanistan IRAC,  impersonated by Pamela mceandry, calling themselves Steve and Mary Female identity change to Paris Pichai or Pichilli Apaguasa. Secured identity with "memory" software accounts of DreamWorks high jacked used against her by known criminals and theives.Stalking of female documented with federal restraining order permanent. Paris 304 663 4110 Sent from my  U.S.Cellular© Smartphone-------- Original message --------From: mc4928184 ***@***.***> Date: 2/28/22  15:53  (GMT-05:00) To: ***@***.*** Subject: FW: Urgent HelpSent from my  U.S.Cellular© Smartphone-------- Original message --------From: mc4928184 ***@***.***> Date: 2/28/22  3:51 PM  (GMT-05:00) To: ***@***.*** Subject: FW: Urgent Data pushesMasked ipPhone info changesAll obsevedSent from my  U.S.Cellular© Smartphone-------- Original message --------From: mc4928184 ***@***.***> Date: 2/24/22  12:07 AM  (GMT-05:00) To: ***@***.*** Subject: Urgent Crime Sent from my  U.S.Cellular© SmartphoneSent from my  U.S.Cellular© Smartphone
-------- Original message --------From: Peter Bengtsson ***@***.***> Date: 3/4/22  11:15  (GMT-05:00) To: github/docs ***@***.***> Cc: Subscribed ***@***.***> Subject: Re: [github/docs] reduce max_old_space_size for jest tests (PR #15975) 
@mikesurowiec Now it's just changing the memory amount.
Still, it doesn't appear to solve much. Just a good practice.

**Comment by MAC-ARTHUR-TRUE** [comment]
Madison chambers or Chamberlin POW MIA Afghanistan IRAC,  impersonated by Pamela mceandry, calling themselves Steve and Mary Female identity change to Paris Pichai or Pichilli Apaguasa. Secured identity with "memory" software accounts of DreamWorks high jacked used against her by known criminals and theives.Stalking of female documented with federal restraining order permanent. Paris 304 663 4110 Sent from my  U.S.Cellular© Smartphone-------- Original message --------From: mc4928184 ***@***.***> Date: 2/28/22  15:53  (GMT-05:00) To: ***@***.*** Subject: FW: Urgent HelpSent from my  U.S.Cellular© Smartphone-------- Original message --------From: mc4928184 ***@***.***> Date: 2/28/22  3:51 PM  (GMT-05:00) To: ***@***.*** Subject: FW: Urgent Data pushesMasked ipPhone info changesAll obsevedSent from my  U.S.Cellular© Smartphone-------- Original message --------From: mc4928184 ***@***.***> Date: 2/24/22  12:07 AM  (GMT-05:00) To: ***@***.*** Subject: Urgent Crime Sent from my  U.S.Cellular© SmartphoneSent from my  U.S.Cellular© Smartphone
-------- Original message --------From: Peter Bengtsson ***@***.***> Date: 3/3/22  16:15  (GMT-05:00) To: github/docs ***@***.***> Cc: Subscribed ***@***.***> Subject: Re: [github/docs] reduce max_old_space_size for jest tests (PR #15975) 
@mikesurowiec It didn't seem to be the silver bullet we had hoped for. But it still seems like the totally right thing to do.

**Review by heiskr — APPROVED** [review]

**Review by xty — COMMENTED** [review]
This seems to have [helped with tests](https://github.com/github/docs/runs/5503075659) on #16181 👍

**Comment by peterbe** [comment]
> This seems to have [helped with tests](https://github.com/github/docs/runs/5503075659) on #16181 👍

Unfortunately, it probably wasn't a silver bullet fix. We were able to see the same `jest` crashes even with this change made.

**Comment by alex00j** [comment]
Хай

File: package.json [hunk 1]
```diff
@@ -210,7 +210,7 @@
     "translation-check": "start-server-and-test translation-check-server 4002 translation-check-test",
     "translation-check-server": "cross-env NODE_ENV=test PORT=4002 node server.mjs",
     "translation-check-test": "script/i18n/test-html-pages.js",
-    "test": "cross-env NODE_OPTIONS='--max_old_space_size=4096 --experimental-vm-modules' jest --logHeapUsage --detectOpenHandles",
+    "test": "cross-env NODE_OPTIONS='--max_old_space_size=4096 --experimental-vm-modules' jest --logHeapUsage",
```
Comment (alex00j) [inline package.json:213]: Дубликат #

**Pull request merged**